config/environments/production.rb

```diff
@@ -75,7 +75,7 @@
   config.active_support.deprecation = :notify
 
   # Use default logging formatter so that PID and timestamp are not suppressed.
-  # config.log_formatter = ::Logger::Formatter.new
+  config.log_formatter = ::Logger::Formatter.new
```

Comment:
Did you look at 98c7f55ac9d62c15e9cac4faa4135ac2df648357 and decide whether or not this should remain uncommented?

([Other apps](https://github.com/alphagov/collections/pull/2287/commits/7a15ec388c4a4920e17ba5ee4752071ed068c06f#diff-da60b4e96eff2b132991226d308949e23f4ef3aad45ad59edd09cbc32cc6251eR84) have it uncommented too, so probably fine - just wondered if this was a deliberate choice).

Reply:
this wasn't a deliberate choice as such, but it seemed innocuous. 
I'm still investigating why this build is failing even after updating, so it's likely to change further and more than happy to re-comment this!